proof/capDL-api/TCB_DP.thy

```diff
@@ -1169,7 +1169,7 @@ lemma restart_cdl_current_domain:
   apply (simp add:restart_def)
   apply (wp alternative_wp)
     apply (simp add:cancel_ipc_def)
-    apply (wpsimp wp: hoare_pre_cont[where a="revoke_cap_simple sl" for sl])+
+    apply (wpsimp wp: hoare_pre_cont[where f="revoke_cap_simple sl" for sl])+
```

Comment:
did this sneak into the wrong commit? it is capDL-api, but there was a commit for hoare_pre_cont variable renamed I think. also again below

Reply:
it probably did, yes. Will see if I can extract it.